charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -511,3 +511,157 @@ instrumentation:
   #   limits:
   #     memory: "128Mi"
   #     cpu: "500m"
+
+# Bridge cluster role configuration
+opAMPBridgeClusterRole:
+  # Whether the bridge cluster role is enabled or not
+  enabled: true
+
+  # Annotations for the bridge cluster role
+  annotations: {}
+
+  # Rules for the bridge cluster role
+  rules: []
+
+# OpAMP bridge configuration
+opAMPBridge:
+  # Whether OpAMP bridge is enabled or not
+  enabled: false
+
+  # Adds `opentelemetry.io/opamp-reporting: true` to all collectors
+  addReportingLabel: true
+  # Adds `opentelemetry.io/opamp-managed: true` to all collectors
+  addManagedLabel: false
+
+  # Endpoint for OpAMP server
+  endpoint: http://opamp-server:8080
+
+  # Headers configuration for OpAMP bridge
+  headers: {}
+  # Authorization: Bearer your_access_token
+  # Custom-Header: Custom-Value
+
+  # Capabilities of OpAMP bridge
+  capabilities:
+    AcceptsOpAMPConnectionSettings: true
+    AcceptsOtherConnectionSettings: true
+    AcceptsRemoteConfig: true
+    AcceptsRestartCommand: true
+    ReportsEffectiveConfig: true
+    ReportsHealth: true
+    ReportsOwnLogs: true
+    ReportsOwnMetrics: true
+    ReportsOwnTraces: true
+    ReportsRemoteConfig: true
+    ReportsStatus: true
+
+  # Components allowed for OpAMP bridge
+  componentsAllowed: {}
+  # receiver:
+  #   - otlp
+  #   - prometheus
+  # processor:
+  #   - batch
+  #   - memory_limiter
+  # exporter:
+  #   - prometheusremotewrite
+
+  # Resources configuration for OpAMP bridge
+  resources:
+    limits:
+      cpu: "250m"
+      memory: "256Mi"
+    requests:
+      cpu: "250m"
+      memory: "256Mi"
+
+  # Number of replicas for OpAMP bridge
+  replicas: 1
```

Comment:
Do we need to support more than 1 replicas?

Reply:
not right now... removing!